### PR TITLE
Add Support for Commands API

### PR DESCRIPTION
**Pull request by kristinpeterson**
Add support for the [Commands API](https://m2x.att.com/developer/documentation/v2/commands).

### CODE REVIEW

**Comment by kristinpeterson** [comment]
@fsaravia @stream7 this is ready for review

File: src/commands.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+define(["helpers"], function(helpers) {
+  // Wrapper for AT&T M2X Commands API
+  //
+  // https://m2x.att.com/developer/documentation/v2/commands
```
Comment (fsaravia) [inline src/commands.js]: Can you align the comment lines with the method definitions? The same in lines 9-13

**Comment by fsaravia** [comment]
Other than the comment above, this looks good!

**Comment by kristinpeterson** [comment]
@fsaravia fixed! Thank you for reviewing!

**Pull request merged**